vyked/registry.py

```diff
@@ -296,12 +296,12 @@ def _connect_to_service(self, host, port, node_id, service_type):
                 coroutine = self._loop.create_connection(partial(get_vyked_protocol, self), host, port)
                 future = asyncio.async(coroutine)
                 future.add_done_callback(partial(self._handle_service_connection, node_id, host, port))
-        elif service_type == 'http':
+        elif service_type == 'ws' or service_type == 'http':
```

Comment:
elif ``service_type in ('ws', 'http')``. Looks more short and concise but that's just my opinion.